crates/bevy_window/src/window.rs

```diff
@@ -720,8 +720,8 @@ impl Window {
         self.focused
     }
     /// Get the [`RawWindowHandleWrapper`] corresponding to this window
```

Comment:
Docs need updating.

Reply:
Updated in https://github.com/bevyengine/bevy/pull/6114/commits/2cccd038dfb900dfac86a13e1bd29bd82d90b2ae